net-irc/inspircd/inspircd-3.0.1.ebuild

```diff
@@ -0,0 +1,133 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit toolchain-funcs user
+
+DESCRIPTION="Inspire IRCd - The Stable, High-Performance Modular IRCd"
+HOMEPAGE="https://inspircd.github.com/"
+SRC_URI="https://github.com/inspircd/inspircd/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE="debug gnutls ldap maxminddb mbedtls mysql pcre posix postgres re2 sqlite ssl sslrehashsignal stdlib tre"
+
+RDEPEND="
+	dev-lang/perl
+	gnutls? ( net-libs/gnutls:= dev-libs/libgcrypt:0 )
+	ldap? ( net-nds/openldap )
+	maxminddb? ( dev-libs/libmaxminddb )
+	mbedtls? ( net-libs/mbedtls:= )
+	mysql? ( dev-db/mysql-connector-c:= )
+	pcre? ( dev-libs/libpcre )
+	postgres? ( dev-db/postgresql:= )
+	re2? ( dev-libs/re2:= )
+	sqlite? ( >=dev-db/sqlite-3.0 )
+	ssl? ( dev-libs/openssl:= )
+	stdlib? ( >=sys-devel/gcc-4.9.0 )
```

Comment:
This dependency is meaningless. Just because you have a new enough GCC installed doesn't mean you're actually using it.

Besides, 4.9.0 is so ancient it's not really worth checking. Our current profiles require at least 6.4.0.

Reply:
Ah, good point.  I nuked it.